src/DssSpell.sol

```diff
@@ -50,42 +61,140 @@ contract DssSpellAction is DssAction {
-        // Note: see `dao_resolutions` variable declared above
+        // UPMaker - 1.85 MKR - 0xbB819DF169670DC71A16F58F55956FE642cc6BcD
+        MKR.transfer(UPMAKER, 1.85 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
+
+        // ---------- Q2 2024 AVC Member Compensation ----------
+        // Forum: https://forum.makerdao.com/t/avc-member-participation-rewards-q2-2024/24442/1
+
+        // IamMeeoh - 12.51 MKR - 0x47f7A5d8D27f259582097E1eE59a07a816982AE9
+        MKR.transfer(IAMMEEOH, 12.51 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
+
+        // DAI-Vinci - 12.51 MKR - 0x9ee47F0f82F1A6F45C4E1D25Ce95C321D8C8356a
+        MKR.transfer(DAI_VINCI, 12.51 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
+
+        // opensky - 12.51 MKR - 0xf44f97f4113759E0a57756bE49C0655d490Cf19F
+        MKR.transfer(OPENSKY_2, 12.51 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
+
+        // Res - 12.51 MKR - 0x8c5c8d76372954922400e4654AF7694e158AB784
+        MKR.transfer(RES, 12.51 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
+
+        // Harmony - 12.51 MKR - 0xE20A2e231215e9b7Aa308463F1A7490b2ECE55D3
+        MKR.transfer(HARMONY_2, 12.51 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
+
+        // Libertas - 12.51 MKR - 0xE1eBfFa01883EF2b4A9f59b587fFf1a5B44dbb2f
+        MKR.transfer(LIBERTAS, 12.51 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
+
+        // seedlatam.eth - 12.51 MKR - 0xd43b89621fFd48A8A51704f85fd0C87CbC0EB299
+        MKR.transfer(SEEDLATAMETH_2, 12.51 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
+
+        // 0xRoot - 12.51 MKR - 0xC74392777443a11Dc26Ce8A3D934370514F38A91
+        MKR.transfer(ROOT, 12.51 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
+
+        // ---------- Launch Project Funding ----------
+        // Forum: https://forum.makerdao.com/t/utilization-of-the-launch-project-under-the-accessibility-scope/21468/17
+
+        // Transfer 5,000,000 DAI to the Launch Project at 0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F
+        DssExecLib.sendPaymentFromSurplusBuffer(LAUNCH_PROJECT_FUNDING, 5_000_000);
+
+        // Transfer 450.00 MKR to the Launch Project at 0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F
+        MKR.transfer(LAUNCH_PROJECT_FUNDING, 450.00 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
 
         // ---------- Spark Spell ----------
-        // Forum: https://forum.makerdao.com/t/may-21-2024-proposed-changes-to-sparklend-for-upcoming-spell/24327
+        // Forum: https://forum.makerdao.com/t/may-31-2024-proposed-changes-to-sparklend-for-upcoming-spell/24413
+        // Vote: https://vote.makerdao.com/polling/QmPmVeDx
```

Comment:
Please correct the prefix to `Poll:` here

Reply:
Thanks, addressed via 054888c